neps/nep-0001.md

```diff
@@ -1,224 +1,170 @@
 
-## Transferring NEP Ownership
-It occasionally becomes necessary to transfer ownership of NEPs to a new author. In general, it is preferable to retain the original author as a co-author of the transferred NEP, but that is up to the original author. A good reason to transfer ownership is because the original author no longer has the time or interest in updating it or following through with the NEP process. A bad reason to transfer ownership is because the author does not agree with the direction of the NEP. One aim of the NEP process is to try to build consensus around a NEP, but if that is not possible, an author can submit a competing NEP.
+### Auxiliary Files
 
-If you are interested in assuming ownership of a NEP, you can also do this via pull request. Fork the NEP repository, make your ownership modification, and submit a pull request. In the PR description, tag the original author and provide a summary of the work that was previously done. Also clearly state the intent of the fork and the relationship of the new PR to the old one. For example: "Forked to address the remaining review comments in NEP # since the original author does not have time to address them."
+Images, diagrams, and auxiliary files should be included in a subdirectory of the assets folder for that NEP as follows: assets/nep-N (where N is to be replaced with the NEP number). When linking to an image in the NEP, use relative links such as .../assets/nep-1/image.png
 
-## Style Guide
+### Style Guide
 
 #### NEP numbers
-When referring to a NEP by number, it should be written in the hyphenated form NEP-X where Xis the NEP’s assigned number.
+
+When referring to a NEP by number, it should be written in the hyphenated form NEP-X where X is the NEP's assigned number.
 
 #### RFC 2119
-NEPs are encouraged to follow [RFC](https://www.ietf.org/rfc/rfc2119.txt)[ 2119](https://www.ietf.org/rfc/rfc2119.txt) for terminology and to insert the following at the beginning of the Specification section:
 
-The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+NEPs are encouraged to follow [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt) for terminology and to insert the following at the beginning of the Specification section:
+
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).  
+
+## NEP Maintenance
+
+In general, NEPs are no longer modified after they have reached the final state.
```

Comment:
Reading it end-to-end once again, I feel this section might be misleading as we actually encourage people to submit NEP extensions (PRs with the changes to the existing NEP documents). Here is the proposed version:

```suggestion
## NEP Maintenance

In general, NEPs are no longer modified after they have reached the final state, but there are occasions when it is preferable to update a NEP. In case of discovering a security vulnerability or identifying a misalignment with the actual widely used implementations, it is better to submit a NEP extension (a pull request with proposed changes to an existing NEP document).

NEP extension will have a high chance of approval if it does not introduce a breaking change and it brings a clear win to the existing implementors of the NEP.

If a NEP extension can be a separate NEP, prefer submitting a new NEP and just specify that it depends on certain NEPs instead of modifying the existing one.
```